src/Toolkit.Forms/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms.csproj

```diff
@@ -86,6 +87,9 @@
     <EmbeddedResource Include="Assets\x-small.png" />
     <EmbeddedResource Include="Assets\x.png" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Assets\calcite-ui-icons-24.ttf" />
```

Comment:
I've updated the PR to only include the font file once, but add it as a link to the UWP project.

I checked and confirmed that the font is only added to the UWP build for Toolkit and it is added as content, so I don't think it can be used in Forms through the shared toolkit.